### PR TITLE
(#566) test case cleanups

### PR DESCRIPTION
**Pull request by rathorevaibhav**
#566 

Changes include:

1. Remove `anAuthorizedUser` as it was confusing.
2. Updated `signIn` method. By default, it'll log in as an employee. 
3. Some cleanups in test files.

### CODE REVIEW

File: tests/TestCase.php [hunk 1]
```diff
@@ -2,15 +2,17 @@
...
 abstract class TestCase extends BaseTestCase
 {
     use CreatesApplication;
 
-    protected function signIn($user = null)
+    protected function signIn($role = 'employee')
     {
-        $user = $user ?: create('App\User');
+        $user = create(User::class);
+        $user->assignRole($role);
```
Comment (pankaj-ag) [inline tests/TestCase.php]: @rathorevaibhav I think default $role should be null. Otherwise, I won't be able to signin without any role.

Comment (rathorevaibhav) [reply tests/TestCase.php]: Not sure. Why do we need to sign-in without any role? There's nothing a person can do in functionality without any role.

Moreover, IMO, anyone who's onboarding will be an employee of an organization. I'm considering employee will be the role with lowest privileges.

Comment (pankaj-ag) [reply tests/TestCase.php]: > There's nothing a person can do in functionality without any role.

How will you test this?

**Comment by rathorevaibhav** [comment]
@pankaj-ag done with the feedback.

Assigning no-role will make sense. For example, if we want to check something that is not accessible if a person is not an employee, we won't be able to test it if we assign the employee role to every user.

**Pull request merged**